benchmarks/dynamo/common.py

```diff
@@ -84,6 +84,22 @@
     "xcit_large_24_p8_224",  # fp64_OOM
 ]
 
+CI_SKIP_AOT_EAGER_DYNAMIC_TRAINING = [
+    *CI_SKIP_AOT_EAGER_TRAINING,
```

Comment:
We haven't updated `CI_SKIP_AOT_EAGER_TRAINING` after the migration. This may be a good chance to add that.

Reply:
Wait, based on my reading of the scripts, we are still applying these skips. So what do you mean by they're not updated?